### DISCUSSION TITLE
Exclude Pages from HTML Export?

### DISCUSSION THREAD

**Discussion by unit-rx55379** [zim-desktop-wiki/zim-desktop-wiki#2650] [Q&A] [answered]
I'm new to Zim and it's literally the perfect program/system for my needs, except that I can't find a way of excluding unfinished pages from the HTML export. Is there a tag or signifier I can use to mark a page as "incomplete" and the exporter either ignores it or generates an "under construction" placeholder? I saw a mention of meta-tags somewhere that implied this would be possible, but I seem to have lost the link.

If it's not possible, is there a custom tool available that I haven't been able to find? And if there is no custom tool, how do I submit a feature request (I'm also new to github...)?

Thanks much.

**Comment by jaap-karssenberg** [comment, marked as answer]
Afraid the current export function does not have such fine grained
selection control. Best work-around would be something like a custom script
that copies a sub-set of your notebook to a temporary folder and run the
export on that.

As a broad outline I have been thinking to allow the search function to
make selections which can be exported. E.g. from the search results add an
"export" button which than exports the pages matching the results only. But
this might have some complexity in implementing it.

A more simple approach would be to have a "exclude" or "include" filter in
the export dialog which runs a simple query on the pages selected for
export.

If you get around to hacking something together, GitHub pull requests are
the way to get changes merged.

Regards,

Jaap

**Reply by unit-rx55379** [reply]
Thanks for getting back to me. I figured out a way of implementing a GTD-style file structure such that finished pieces land in a "publish" folder that I then export from. Ultimately, it's probably a better organisation than I would have implemented if there *was* a working tag...

"A more simple approach would be to have a "exclude" or "include" filter in
the export dialog which runs a simple query on the pages selected for
export."

This is more or less what I was thinking. Just an "@exclude" tag or equivalent that causes the exporter to skip that particular iteration. I did have a look into the source code to see if I could figure out what was going on, but I have to admit that I'm not much of a programmer, and certainly not programmer enough to tinker without unwittingly breaking something important :)
